.github/workflows/trivy-jira.yml

```diff
@@ -0,0 +1,61 @@
+# Run trivy creating a github issue for any issues
+# For each github issue create a Jira Ticket linking to the issue
+name: Trivy Jira
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 4 * * MON'
+
+env:
+  REGISTRY: dataswift/hat
+
+  jobs:
+    trivy-scan:
+      name: Scan
+      runs-on: ubuntu-latest
+      timeout-minutes: 20
+      steps:
+
+jobs:
+  trivy-scan:
+    name: Launch Trivy Scan
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    steps:
+      - name: Container - Login
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_DATASWIFT_USER }}
+          password: ${{ secrets.DOCKERHUB_DATASWIFT_PASS }}
+
+      - name: Container - Pull
+        run: docker pull ${{ env.ECR_REGISTRY }}:${{ github.sha }}
+
+      - name: Container - Scan
+        uses: dataswift/gha-trivy@main
```

Comment:
@pperzyna can i use this from an open source project ?

Reply:
yes, you can